### ISSUE TITLE
运行web_ui.py报错：Segmentation fault (core dumped)

### ISSUE THREAD

**Issue by smallflyingpig** [FunAudioLLM/CosyVoice#74]
**Describe the bug**
2024-07-09 21:49:45,485 - modelscope - INFO - PyTorch version 2.3.0+cu118 Found.
2024-07-09 21:49:45,488 - modelscope - INFO - Loading ast index from /home/hadoop-hmart-peisongpa/.cache/modelscope/ast_indexer
2024-07-09 21:49:46,901 - modelscope - INFO - Loading done! Current index file version is 1.15.0, with md5 89be555b4b57c4a68d2b1c52b4b2f4f9 and a total number of 980 components indexed
/home/hadoop-hmart-peisongpa/dolphinfs_hdd_hadoop-hmart-peisongpa/lijiguo/conda/envs/fun_audio_llm38/lib/python3.8/site-packages/torch/_jit_internal.py:739: FutureWarning: ignore(True) has been deprecated. TorchScript will now drop the function call on compilation. Use torch.jit.unused now. {}
  warnings.warn(
/home/hadoop-hmart-peisongpa/dolphinfs_hdd_hadoop-hmart-peisongpa/lijiguo/conda/envs/fun_audio_llm38/lib/python3.8/site-packages/diffusers/models/lora.py:393: FutureWarning: `LoRACompatibleLinear` is deprecated and will be removed in version 1.0.0. Use of `LoRACompatibleLinear` is deprecated. Please switch to PEFT backend by installing PEFT: `pip install peft`.
  deprecate("LoRACompatibleLinear", "1.0.0", deprecation_message)
2024-07-09 21:50:07,017 INFO input frame rate=50
/home/hadoop-hmart-peisongpa/dolphinfs_hdd_hadoop-hmart-peisongpa/lijiguo/conda/envs/fun_audio_llm38/lib/python3.8/site-packages/torch/nn/utils/weight_norm.py:28: UserWarning: torch.nn.utils.weight_norm is deprecated in favor of torch.nn.utils.parametrizations.weight_norm.
  warnings.warn("torch.nn.utils.weight_norm is deprecated in favor of torch.nn.utils.parametrizations.weight_norm.")
2024-07-09 21:50:13.197957298 [W:onnxruntime:, session_state.cc:1162 VerifyEachNodeIsAssignedToAnEp] Some nodes were not assigned to the preferred execution providers which may or may not have an negative impact on performance. e.g. ORT explicitly assigns shape related ops to CPU to improve perf.
2024-07-09 21:50:13.198013075 [W:onnxruntime:, session_state.cc:1164 VerifyEachNodeIsAssignedToAnEp] Rerunning with verbose output on a non-minimal build will show node assignments.
/mnt/dolphinfs/hdd_pool/docker/user/hadoop-hmart-peisongpa/lijiguo/workspace/FunAudioLLM-APP/cosyvoice/cosyvoice/cli/../../../../../data/models/speech_kantts_ttsfrd/resource
load leagacy transf breakmodel
load leagacy transf breakmodel
Segmentation fault (core dumped)

**Desktop (please complete the following information):**
 - OS: centos 8
 - Python: 3.8 

**Pip list**
absl-py                   2.1.0
addict                    2.4.0
aiofiles                  23.2.1
aiohttp                   3.9.5
aiosignal                 1.3.1
aliyun-python-sdk-core    2.15.1
aliyun-python-sdk-kms     2.16.3
altair                    5.3.0
annotated-types           0.7.0
antlr4-python3-runtime    4.9.3
anyio                     4.4.0
async-timeout             4.0.3
attrs                     23.2.0
audioread                 3.0.1
beautifulsoup4            4.12.3
cachetools                5.3.3
certifi                   2024.7.4
cffi                      1.16.0
charset-normalizer        3.3.2
click                     8.1.7
coloredlogs               15.0.1
conformer                 0.3.2
contourpy                 1.1.1
crcmod                    1.7
cryptography              42.0.8
cycler                    0.12.1
dashscope                 1.20.1
datasets                  2.18.0
decorator                 5.1.1
deepspeed                 0.14.2
diffusers                 0.27.2
dill                      0.3.8
dnspython                 2.6.1
editdistance              0.8.1
einops                    0.8.0
email_validator           2.2.0
exceptiongroup            1.2.1
fastapi                   0.111.0
fastapi-cli               0.0.4
ffmpy                     0.3.2
filelock                  3.15.4
flatbuffers               24.3.25
fonttools                 4.53.1
frozenlist                1.4.1
fsspec                    2024.2.0
funasr                    1.1.0
gast                      0.6.0
gdown                     5.1.0
google-auth               2.31.0
google-auth-oauthlib      1.0.0
gradio                    4.32.2
gradio_client             0.17.0
grpcio                    1.57.0
grpcio-tools              1.57.0
h11                       0.14.0
hjson                     3.1.0
httpcore                  1.0.5
httptools                 0.6.1
httpx                     0.27.0
huggingface-hub           0.23.4
humanfriendly             10.0
Hydra                     2.5
hydra-core                1.3.2
HyperPyYAML               1.2.2
idna                      3.7
importlib_metadata        8.0.0
importlib_resources       6.4.0
inflect                   7.3.1
jaconv                    0.3.4
jamo                      0.4.1
jieba                     0.42.1
Jinja2                    3.1.4
jmespath                  0.10.0
joblib                    1.4.2
jsonschema                4.22.0
jsonschema-specifications 2023.12.1
kaldiio                   2.18.0
kiwisolver                1.4.5
lazy_loader               0.4
librosa                   0.10.2
lightning                 2.2.4
lightning-utilities       0.11.3.post0
llvmlite                  0.41.1
Markdown                  3.6
markdown-it-py            3.0.0
MarkupSafe                2.1.5
matplotlib                3.7.5
mdurl                     0.1.2
modelscope                1.15.0
more-itertools            10.3.0
mpmath                    1.3.0
msgpack                   1.0.8
multidict                 6.0.5
multiprocess              0.70.16
networkx                  3.1
ninja                     1.11.1.1
numba                     0.58.1
numpy                     1.24.4
nvidia-cublas-cu11        11.11.3.6
nvidia-cuda-cupti-cu11    11.8.87
nvidia-cuda-nvrtc-cu11    11.8.89
nvidia-cuda-runtime-cu11  11.8.89
nvidia-cudnn-cu11         8.7.0.84
nvidia-cufft-cu11         10.9.0.58
nvidia-curand-cu11        10.3.0.86
nvidia-cusolver-cu11      11.4.1.48
nvidia-cusparse-cu11      11.7.5.86
nvidia-nccl-cu11          2.20.5
nvidia-nvtx-cu11          11.8.86
oauthlib                  3.2.2
omegaconf                 2.3.0
onnxruntime-gpu           1.16.0
openai-whisper            20231117
orjson                    3.10.6
oss2                      2.18.6
packaging                 24.1
pandas                    2.0.3
pillow                    10.4.0
pip                       24.0
pkgutil_resolve_name      1.3.10
platformdirs              4.2.2
pooch                     1.8.2
protobuf                  4.25.0
psutil                    6.0.0
py-cpuinfo                9.0.0
pyarrow                   16.1.0
pyarrow-hotfix            0.6
pyasn1                    0.6.0
pyasn1_modules            0.4.0
pycparser                 2.22
pycryptodome              3.20.0
pydantic                  2.7.0
pydantic_core             2.18.1
pydub                     0.25.1
Pygments                  2.18.0
pynndescent               0.5.13
pynvml                    11.5.0
pyparsing                 3.1.2
PySocks                   1.7.1
python-dateutil           2.9.0.post0
python-dotenv             1.0.1
python-multipart          0.0.9
pytorch-lightning         2.3.2
pytorch-wpe               0.0.1
pytz                      2024.1
PyYAML                    6.0.1
referencing               0.35.1
regex                     2024.5.15
requests                  2.32.3
requests-oauthlib         2.0.0
rich                      13.7.1
rotary-embedding-torch    0.6.4
rpds-py                   0.19.0
rsa                       4.9
ruamel.yaml               0.18.6
ruamel.yaml.clib          0.2.8
ruff                      0.5.1
safetensors               0.4.3
scikit-learn              1.3.2
scipy                     1.10.1
semantic-version          2.10.0
sentencepiece             0.2.0
setuptools                69.5.1
shellingham               1.5.4
simplejson                3.19.2
six                       1.16.0
sniffio                   1.3.1
sortedcontainers          2.4.0
soundfile                 0.12.1
soupsieve                 2.5
soxr                      0.3.7
starlette                 0.37.2
sympy                     1.12.1
tensorboard               2.14.0
tensorboard-data-server   0.7.2
tensorboardX              2.6.2.2
threadpoolctl             3.5.0
tiktoken                  0.7.0
tokenizers                0.19.1
tomli                     2.0.1
tomlkit                   0.12.0
toolz                     0.12.1
torch                     2.3.0+cu118
torch-complex             0.4.4
torchaudio                2.3.0+cu118
torchmetrics              1.4.0.post0
tqdm                      4.66.4
transformers              4.42.3
triton                    2.3.0
ttsfrd                    0.3.6
typeguard                 4.3.0
typer                     0.12.3
typing_extensions         4.12.2
tzdata                    2024.1
ujson                     5.10.0
umap-learn                0.5.6
urllib3                   2.2.2
uvicorn                   0.30.1
uvloop                    0.19.0
watchfiles                0.22.0
websocket-client          1.8.0
websockets                11.0.3
Werkzeug                  3.0.3
wget                      3.2
wheel                     0.43.0
xxhash                    3.4.1
yapf                      0.40.2

**Comment by aluminumbox** [comment]
it seems to be a ttsfrd fault, update code and install requirements again, we use wetextprocessing instead

**Comment by ChaoliesTinnu** [comment]
我也是，移除ttsfrd就好了
报错：
`
[E:onnxruntime:Default, provider_bridge_ort.cc:1480 TryGetProviderInfo_CUDA] /onnxruntime_src/onnxruntime/core/session/provider_bridge_ort.cc:1193 onnxruntime::Provider& onnxruntime::ProviderLibrary::Get() [ONNXRuntimeError] : 1 : FAIL : Failed to load library libonnxruntime_providers_cuda.so with error: libcublasLt.so.11: cannot open shared object file: No such file or directory

2024-07-16 09:14:38.819616148 [W:onnxruntime:Default, onnxruntime_pybind_state.cc:743 CreateExecutionProviderInstance] Failed to create CUDAExecutionProvider. Please reference https://onnxruntime.ai/docs/execution-providers/CUDA-ExecutionProvider.html#requirements to ensure all dependencies are met.
非法指令 (核心已转储)
`
移除ttsfrd，使用wetextprocessing，会报一些warming
```
pip list | grep ttsfrd
pip3 uninstall ttsfrd
```
然后就可以正常启动

**Comment by zhaoshuyan** [comment]
我已移除了ttsfrd,复刻声音时还是会报Segmentation fault (core dumped)
